<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportExportGUI_1Sel1LineEdit2ListWidget_QTD</class>
 <widget class="QWidget" name="ImportExportGUI_1Sel1LineEdit2ListWidget_QTD">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>222</width>
    <height>178</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="GroupBox1">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout">
      <property name="margin">
       <number>9</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="LineEdit2"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="TextLabel2">
        <property name="text">
         <string>TL2</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="PushButton1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="TextLabel1">
        <property name="text">
         <string>TL1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="LineEdit1">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="TextLabel3">
        <property name="text">
         <string>TL3</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="TextLabel4">
        <property name="text">
         <string>TL4</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QListWidget" name="listWidget1"/>
      </item>
      <item row="3" column="2">
       <widget class="QListWidget" name="listWidget2"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>PushButton1</tabstop>
  <tabstop>LineEdit1</tabstop>
  <tabstop>LineEdit2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
